src/content/en/fundamentals/progressive-web-apps/prpl.md

```diff
@@ -0,0 +1,242 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: 
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+Caution: PRPL is a pattern we believe makes sense for applications that can be
```

Comment:
@petele / @jpmedley would you mind if we switched this over to a note per Gray's feedback? I really wish devsite had something closer to an experimental callout. I think that would address the first piece of feedback here.

@graynorton I think my phrasing here may have been inspired by some conversations with Matt about PRPL being useful for a particular class of applications. Perhaps this wording might be better? "PRPL is a new pattern we feel has great potential. At this stage, we welcome experimentation with it while we iterate on the ideas in the pattern and collect more data on where it offers the greatest benefits."